pkg/plugins/backendplugin/instrumentation.go

```diff
@@ -0,0 +1,40 @@
+package backendplugin
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	pluginRequestCounter *prometheus.CounterVec
+	pluginRequestLatency *prometheus.SummaryVec
+)
+
+func init() {
+	pluginRequestCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "plugin_request_total",
+		Help: "The total amount of plugin requests",
+	}, []string{"name", "endpoint", "status"})
+
+	pluginRequestLatency = prometheus.NewSummaryVec(prometheus.SummaryOpts{
+		Name:       "plugin_request_latency",
```

Comment:
Should this be named similar to existing `http_request_duration_milliseconds` like `plugin_request_duration_milliseconds`?

Reply:
argh. Yes Forgot that we use `ms` due to the graphite bridge.

Reply:
Should be `_seconds` otherwise :) good catch

Reply:
Why latency instead of duration? Isn't it better to comply with our existing convention/prometheus somewhat standard convention?

Reply:
no good reason at all. I'll fix this. Anything else I should address? I dont want to ping pong this PR to long.